src/mock.ts

```diff
@@ -65,7 +65,7 @@ export function ergonomock(
     throw new Error("Ergonomock requires a GraphQL query, either as a string or DocumentNode.");
   }
 
-  const document = typeof query === "string" ? parse(query) : query;
+  const document = addTypenameToDocument(typeof query === "string" ? parse(query) : query);
```

Comment:
Should we make this conditional based on an `options` key the same way that MockLink does here https://github.com/apollographql/apollo-client/blob/f49890005a491db10817916361818dce078f7036/src/utilities/testing/mocking/mockLink.ts#L33 ?

Reply:
Actually, we could also move it up one level to https://github.com/SurveyMonkey/graphql-ergonomock/blob/master/src/apollo/MockLink.ts#L50 instead, since MockLink already takes `addTypename`

Reply:
Can you also make sure we have a test with and without addTypename to prevent regressions on that behavior?